### PR TITLE
Closes #27854: Remove unnecessary Theme.getTheme() calls from FirefoxTheme usages

### LINKED ISSUE #27854: Remove all unnecessary `Theme.getTheme()` calls from usages of `FirefoxTheme`
We have a lot of instances of `FirefoxTheme(theme = Theme.getTheme()) { ... }` when the default parameter for `FirefoxTheme.theme` is already `Theme.getTheme()`, making this instantiation redundant. All of these uses should be cleaned up to remove noise from the Compose code.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21993)

### PR DESCRIPTION
**Pull request by BFadairo**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #27854

### CODE REVIEW

**Comment by MozillaNoah** [comment]
Thanks for the contribution, @BFadairo! This is looking great! I'm still seeing ~18 instances of `theme = Theme.getTheme()` that can be removed. Could you please include them with your changes here? Thanks again!

**Comment by MozillaNoah** [comment]
Thanks for the quick turnaround with that update! It looks like there may have been a find & replace error. There's a trailing parenthetical on `FirefoxTheme`:

`FirefoxTheme) {`

**Comment by BFadairo** [comment]
> Thanks for the quick turnaround with that update! It looks like there may have been a find & replace error. There's a trailing parenthetical on `FirefoxTheme`:
> 
> `FirefoxTheme) {`

Thanks! I went ahead and fixed those issues

**Comment by MozillaNoah** [comment]
Could you run the linter over your branch? I think some imports need to be cleaned up.

`./gradlew ktlint detekt`

**Comment by BFadairo** [comment]
> Could you run the linter over your branch? I think some imports need to be cleaned up.
> 
> `./gradlew ktlint detekt`

Can do! Just updated!

File: app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt [hunk 1]
```diff
@@ -24,7 +24,6 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.foundation.rememberScrollState
 import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.foundation.verticalScroll
```
Comment (MozillaNoah) [inline app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt:27]: Was this import removed on accident?

Comment (BFadairo) [reply app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt:27]: Yea not sure what is going on today lol. I'll fix that

**Review by MozillaNoah — APPROVED** [review]
This looks great! Let's squash the commits into one singular commit (using the commit message of your original commit), and we can land this on Monday!

**Comment by BFadairo** [comment]
> This looks great! Let's squash the commits into one singular commit (using the commit message of your original commit), and we can land this on Monday!

Will do, Thanks!

**Comment by MozillaNoah** [comment]
@BFadairo this has officially landed! Thanks again for your contribution!

**Pull request merged**